lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -51,6 +63,19 @@ def reload_file(path)
     load path
   end
 
+  def reload_diff_files
+    unless git?
+      return
+    end
+    output, status = Open3.capture2e *%w(git diff --name-only)
```

Comment:
One more thing, in case this is being run from outside the install dir, we should add `, chdir: Msf::Config.install_root` to the capture call to make sure we invoke `git` where it can read the index.

Reply:
This. Absolutely.